spec/errors.md

```diff
@@ -24,9 +24,16 @@ or _Message Function Errors_ in _expressions_ that are not otherwise used by the
 such as _placeholders_ in unselected _patterns_
 or _declarations_ that are never referenced during _formatting_.
 
-In all cases, when encountering a runtime error,
-a message formatter MUST provide some representation of the message.
-An informative error or errors MUST also be separately provided.
+When formatting a message with one or more errors:
+- An implementation MUST provide a way for a user to be informed
+  of the name of at least one of the errors,
+  either directly or via an identifying error code.
```

Comment:
I'm struggling with a few things here.

I think the "name of" part is a bit circuitous. I think the "identifying error code" part is just an example (out of many) that an implementation might employ. We don't require that implementations use the exact names, I think? That way, some implementations can have a `RESOLUTION_ERROR` and others a `ResolutionError` type, usw.

```suggestion
- An implementation MUST provide a mechanism to discover and identify
  at least one of the errors. 
  The exact form of error signaling is implementation defined.
  Some examples include throwing an exception,
  returning an error code, 
  or providing a function or method for enumerating any errors.
```

Reply:
> We don't require that implementations use the exact names, I think?

We kind of do, because our conformance tests use [error codes](https://github.com/unicode-org/message-format-wg/blob/bcda078ecf59a78b7a02e8eeef1bb5406625c3b7/test/tests/functions/number.json#L25).

But I'm fine with the suggested change, which avoids the issue.